microservices-swift/Jenkinsfile

```diff
@@ -0,0 +1,6 @@
+#!groovy
```

Comment:
Probably don't need this

Reply:
f83a03b414306be44258998ad446b86d505d0cfd